sktime/covariance/covariance.py

```diff
@@ -154,15 +173,15 @@ class Covariance(Estimator):
         :filter: docname in docnames
         :keyprefix: covariance-
     """
-    def __init__(self, lagtime: Optional[int] = None, compute_c00: bool = True, compute_c0t: bool = False,
+    def __init__(self, lagtime: int = 0, compute_c00: bool = True, compute_c0t: bool = False,
```

Comment:
what's a lag time of zero mean?

Reply:
means no time lag, this fits together with the default of `compute_c0t` which is set to `False`

Reply:
i meant more in the physical interpretation

Reply:
sorry but I am not sure how to answer this?

Reply:
i think it is not relevant. i was more thinking of like if you were to do a vampnet (or autoencoder thing) where your data timeshift was zero. but that is irrelevant here i was getting mixed up